packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -396,7 +396,9 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
       nextChildren = null;
 
       if (enableProfilerTimer) {
-        stopBaseRenderTimerIfRunning();
+        if (workInProgress.mode & ProfileMode) {
```

Comment:
Why do you need to conditionally stop it? Stopping something already stopped is as cheap as checking the flag, probably cheaper, right?

Reply:
Totally. This is why I had the question in the PR description:
> Should I remove the conditionals around stopBaseRenderTimerIfRunning? (Is it worth having them?)